edx_lint/files/common_constraints.txt

```diff
@@ -31,3 +31,9 @@ django-simple-history==3.0.0
 # So we need to pin it globally, for now.
 # Ticket for unpinning: https://github.com/openedx/edx-lint/issues/407
 importlib-metadata<7
+
+# Cause: https://github.com/openedx/event-tracking/pull/290
+# event-tracking 2.4.1 upgrades to pymongo 4.4.0 which is not supported on edx-platform.
+# We will pin event-tracking to do not break existing installations
+# See
```

Comment:
```suggestion
# This can be unpinned once https://github.com/openedx/edx-platform/issues/34586 has been resolved and edx-platform is running with pymong>= 4.4.0
```